backend/src/routes/groups-routes.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import {createGroup, retrieveGroups, getSpecificGroup, updateName, deleteGroup} from '../controllers/groups-controller';
+import {body} from "express-validator";
+
+const router = express.Router();
+
+router.post('/', [
+    body('name').isString().withMessage('Name must be a string').notEmpty().withMessage('Name is required').isLength({ min: 3, max: 25 })], createGroup);
+router.get('/', retrieveGroups);
+router.get('/:_id', getSpecificGroup);
+router.put('/:_id', [
+    body('name').isString().withMessage('Name must be a string').notEmpty().withMessage('Name is required').isLength({ min: 3, max: 25 })], updateName);
```

Comment:
Validate `newName` instead of `name` -- fix this & the PR will be ready for merging 👍

Reply:
Since this route takes `newName` and not `name` but still requires `name`